src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -402,6 +403,33 @@ export const jordan: LocationsType = {
     has_map: false,
 }
 
+export const singapore: LocationsType = {
+    name: 'singapore',
+    continent: 'asia',
+    display_continent: 'Asia',
+    link: '/careers/locations/singapore/',
+    display_name: 'Singapore',
+    img_alt: 'Singapore alt img',
```

Comment:
This alt text doesn't seem right. Please speak with Swati for the alt text pattern of these images